add-on/chrome/bootstrap.js

```diff
@@ -1226,6 +1228,9 @@ var WindowListener = {
       // remove it from existing tabs (there's no way to do that).
       window.LoopUI.mm.removeDelayedFrameScript(FRAME_SCRIPT);
 
+      let loopMessageManager = window.getGroupMessageManager("loop");
+      loopMessageManager.removeDelayedFrameScript("chrome://browser/content/content.js");
```

Comment:
What if we move the loop frame script to a constant like we're doing with the general message manager `FRAME_SCRIPT`?